examples/lm1b/train.py

```diff
@@ -527,10 +556,11 @@ def encode_strings(strs, max_len):
 
       # Shard data to devices and do a training step.
       with jax.profiler.StepTraceAnnotation("train", step_num=step):
-        batch = common_utils.shard(
-            jax.tree_util.tree_map(np.asarray, next(train_iter))
+        batch = next(train_iter)
+        batch = jax.tree_map(lambda x: jnp.array(x), batch)
+        state, metrics = p_train_step(
+            state, batch, train_config, learning_rate_fn, 0.0, dropout_rngs
```

Comment:
why do you pass 0.0 here? isn't this the default value? also I would always use kwarg here for a number.

Reply:
If I try:
```
state, metrics = jit_train_step(
            state, batch, train_config, learning_rate_fn, dropout_rng=dropout_rngs
        )
```
I get:
```
File "/home/marcuschiam/flax/examples/lm1b/train.py", line 565, in train_and_evaluate
    state, metrics = jit_train_step(
ValueError: pjit does not support kwargs when in_shardings is specified.
```

Reply:
Hmm ok, thanks for letting me know.